README.md

```diff
@@ -545,6 +545,31 @@
   # Good
   result = String.strip(input)
   ```
+  
+* <a name="anonymous-pipeline"></a>
+  Every time you use the "anonymous function in the pipeline" trick a puppy dies!
```

Comment:
I am personally a bit against the funny bit here and I'd just go with "Don't use anonymous bla bla bla". Also many users won't know what the *"anonymous function in the pipeline" trick* is, so I would rather say again just "Don't use anonymous functions in pipelines." or something similar.

Reply:
@michalmuskala ping.